improver/feels_like_temperature.py

```diff
@@ -133,70 +132,58 @@ def calculate_apparent_temperature(temperature, wind_speed,
     the WetBulbTemperature plugin which makes use of the Goff-Gratch method.
 
     Args:
-      temperature (iris.cube.Cube):
-        Cube of air temperatures
-
-      wind_speed (iris.cube.Cube):
-        Cube of 10m wind speeds
-
-      relative_humidity (iris.cube.Cube):
-        Cube of relative humidities
-
-      pressure (iris.cube.Cube):
-        Cube of air pressure
+        temperature (iris.cube.Cube):
+            Cube of air temperatures
+        wind_speed (iris.cube.Cube):
+            Cube of 10m wind speeds
+        relative_humidity (iris.cube.Cube):
+            Cube of relative humidities
+        pressure (iris.cube.Cube):
+            Cube of air pressure
 
     Returns:
-      iris.cube.Cube:
-        Cube of apparent temperatures. The units of apparent temperature
-        will be the same as the units of the temperature cube when it is input
-        into the function.
+        iris.cube.Cube:
+            Cube of apparent temperatures. The units of apparent temperature
+            will be the same as the units of the input temperature cube.
```

Comment:
This sounds good throughout.  I resisted wider refactoring of this module as I thought it would make it horrendous to review.  But I will make the apparent temperature and wind chill functions return arrays, since you have suggested it.